alot/commands/thread.py

```diff
@@ -731,7 +731,7 @@ def apply(self, ui):
         if self.shell:
             self.cmd = [' '.join(self.cmd)]
 
-        # do teh monkey
+        # do the monkey
```

Comment:
this one was actually on purpose:-)

Reply:
you approved, so should I change this back or just merge like it is now?